recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "arcticdb" %}
-{% set version = "2.0.0" %}
+{% set version = "3.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 149660905f2a1169a8bfeeb0f58e35e496747589a7d2147252181c4e826b3683
+  sha256: 592e9a66f2b991de095f83c3fca7cf9934d44315c38528abfd8b3c7a45381fac
```

Comment:
Force pushed new tag - will need to update the sha later

Reply:
I also probably need to include the entrypoint somehow... https://github.com/man-group/ArcticDB/blob/master/setup.cfg#L132

Reply:
For entry-points, according to [the documentation of `conda-build`](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#python-entry-points) (which used to build conda-forge feedstocks' recipe), reusing the same specification should be sufficient.